ngraph/test/util/CMakeLists.txt

```diff
@@ -36,6 +36,5 @@ if(NGRAPH_LIB_VERSIONING_ENABLE)
 endif()
 target_include_directories(ngraph_test_util PRIVATE ${CMAKE_CURRENT_SOURCE_DIR}/.. ${IE_MAIN_SOURCE_DIR}/include)
 target_link_libraries(ngraph_test_util PRIVATE ngraph ngraph_backend libgtest)
-add_dependencies(ngraph_test_util ext_gtest)
```

Comment:
`ext_gtest` is external cmake project. I suppose if we remove such dependency, `make ngraph_test_util` will not build gtest

Reply:
I verified that `make ngraph_test_util` does indeed build gtest. The reason for that is the preceding line. A `target_link_libraries` not only changes the linker invocation, it also adds a dependency. Thus the `libgtest` target is built before linking `ngraph_test_util`. The `libgtest` target depends on the `ext_gtest` dependency. Therefore duplicating it here is redundant.

Reply:
But will all external projects steps are executed? Does it work from clean state?

Reply:
Yes, the remaining dependencies ensure that the external project is still built. It does work from clean state. Also note that the CI performs clean builds and it did succeed.
Note that this PR does not (yet) ask for using a system copy of gtest. Getting there is a little more involved, because OpenVINO actually has two vendor copies of gtest. When using a system copy, `external_gtest.cmake` would be entirely skipped. That's not what happens here though.